### PR TITLE
Feature/cf waf alarms

### PR DESCRIPTION
**Pull request by runningman84**

### CODE REVIEW

File: rules/cloudfront.yaml [hunk 1]
```diff
@@ -30,3 +30,13 @@ spec:
       labels:
         rule_source: opswatch-prometheus-rules
         severity: info
+    - alert: AwsCloudwatchCloudfrontTrafficSurge
+      annotations:
+        summary: Cloudfront request rate increase by a lot within the last 2 hours
+        description: Cloudfront request rate increase is {{$value}} for cloudfront distributionid {{ $labels.dimension_DistributionId }} ({{ $labels.cloudfront_first_alias }}) in customer {{ $labels.overwrite_asy_customer }} account {{ $labels.overwrite_aws_account_id }} region {{ $labels.overwrite_aws_region }}
+        runbook_url: https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/troubleshooting-distributions.html
+      expr: increase(aws_cloudwatch_CloudFront_Requests_DistributionId_Region{}[2h]) > 5000
```
Comment (autarchprinceps) [inline rules/cloudfront.yaml:38]: @runningman84 Is this added in the stream whitelist?

File: rules/documentdb.yaml [hunk 2]
```diff
@@ -1 +1,32 @@
...
+kind: PrometheusRule
+metadata:
+  labels:
+    release: prometheus
+    app: prometheus-operator
+  name: opswatch.documentdb.rules
+  namespace: monitoring
+spec:
+  groups:
+    - name: documentdb.rules
```
Comment (autarchprinceps) [inline rules/documentdb.yaml:11]: Linting missing:
11:5 [indentation] wrong indentation: expected 2 but found 4

File: rules/waf.yaml [hunk 3]
```diff
@@ -32,8 +32,17 @@ spec:
...
       expr: round(rate(aws_cloudwatch_WAFV2_BlockedRequests_Region_Rule_WebACL_sum{dimension_Rule="ALL"}[15m]) / (rate(aws_cloudwatch_WAFV2_AllowedRequests_Region_Rule_WebACL_sum{dimension_Rule="ALL"}[15m]) + rate(aws_cloudwatch_WAFV2_CountedRequests_Region_Rule_WebACL_sum{dimension_Rule="ALL"}[15m]) + rate(aws_cloudwatch_WAFV2_BlockedRequests_Region_Rule_WebACL_sum{dimension_Rule="ALL"}[15m])) * 100) > 20
       for: 30m
       labels:
         severity: warning
+    - alert: AwsCloudwatchWafV2BlockedAttacks
+      annotations:
+        summary: WAF blocked attacks
+        description: WAF blocked {{ $labels.dimension_Attack }} attack for WAF ACL {{ $labels.dimension_WebACL }} in customer {{ $labels.overwrite_asy_customer }} account {{ $labels.overwrite_aws_account_id }} region {{ $labels.overwrite_aws_region }}
+        runbook_url: https://repost.aws/knowledge-center/waf-mitigate-ddos-attacks
+      expr: aws_cloudwatch_WAFV2_BlockedRequests_Attack_WebACL > 10
```
Comment (autarchprinceps) [inline rules/waf.yaml:45]: Dito, is this added in the whitelist. Either way, it would require a stream stack update across all envs and regions

**Review by autarchprinceps — CHANGES_REQUESTED** [review]

File: rules/cloudfront.yaml (hunk 1 above)
Comment (runningman84) [reply rules/cloudfront.yaml]: not yet

File: rules/documentdb.yaml (hunk 2 above)
Comment (runningman84) [reply rules/documentdb.yaml]: fixed

**Pull request merged**